tests/unit/io/Cereal_unittest.cc

```diff
@@ -2,38 +2,55 @@
 #include <shogun/base/Parameter.h>
 #include <gtest/gtest.h>
 
+#include <fstream>
+#include <cstdio>
+
 #include <shogun/lib/SGVector.h>
 #include <cereal/archives/json.hpp>
-#include <fstream>
 
 using namespace shogun;
 
 TEST(Cereal, Json_SGVector_load_equals_saved)
 {
```

Comment:
@vigsterkr I added `tmpnam` and `std::ios_base::failure` to this test. Do you want to have a look? It doesn't look very pleasant but it's working at local (I'll try to figure out why travis cannot find cereal files)